### DISCUSSION TITLE
General Feedback Thread

### DISCUSSION THREAD

**Discussion by Caleb-Cohen** [Together-100Devs/Together#118] [General]
This discussion thread should serve as a starting discussion for new features, fixes, and general feedback surrounding Together.

**Comment by intelagense** [comment]
![image](https://user-images.githubusercontent.com/18331781/210631627-76f31f3d-faa1-47a2-a1dd-c624a84e2e5f.png)

Here is a grayscale visualization of the contrast being low on the orange text. I think we will have this issue in some of the new modals as well that have orange accent text.

**Reply by Caleb-Cohen** [reply]
@Natashka333
